packages/flutter/lib/src/painting/text_style.dart

```diff
@@ -137,7 +144,8 @@ class TextStyle {
       fontWeight: fontWeight,
       fontStyle: fontStyle,
       fontFamily: fontFamily,
-      fontSize: fontSize
+      fontSize: fontSize,
+      letterSpacing: letterSpacing
```

Comment:
Yeah, the order of the fields is different in TextStyle and ui.TextStyle.  I've matched the callee order in each case.